blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpMultiRollupsQueryHandlerIntegrationTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2013-2015 Rackspace
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.rackspacecloud.blueflood.outputs.handlers;
+
+import com.rackspacecloud.blueflood.http.HttpIntegrationTestBase;
+import org.apache.http.HttpResponse;
+import org.apache.http.util.EntityUtils;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class HttpMultiRollupsQueryHandlerIntegrationTest extends HttpIntegrationTestBase {
+    private final long fromTime = 1389124830L;
```

Comment:
Actually they are both in milisecond since epoch.  The fromTime (1389124830) is "Tue, 07 Jan 2014 20:00:30 GMT" and toTime (1439231325000) is "Mon, 10 Aug 2015 18:28:45 GMT".